src/watering_sys.py

```diff
@@ -0,0 +1,24 @@
+from random import random
```

Comment:
Rewrite the subsystem to follow the project general structure - separated into 3 entities: Sensor (in this case 2 sensors, Water level and Capacitive moisture sensor), Analyser (In this case maybe even 2 Analysers?), Actuator (Water pump). Use pubsub communication in all. Please read the issues written up before coding, as it saves time and will address most of your uncertainties! #26 

Look at previous PR if still confused, or ask me or @angela24680403 if need clarification: https://github.com/Willmish/comp0016-farmbeats/pull/21